<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>621</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>120</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,8">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_Superior">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>42</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
background-color:rgb(0,0,0);
}

QPushButton{
background-color:#000000ff;
border-radius:20px;
}

QPushButton:hover{
background-color:rgb(61,61,61);
border-radius:20px;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btn_Menu">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../../Downloads/barra-de-menus.png</normaloff>../../../Downloads/barra-de-menus.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>295</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../../Downloads/boton-menos.png</normaloff>../../../Downloads/boton-menos.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_Cerrar">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../../Downloads/boton-cerrar.png</normaloff>../../../Downloads/boton-cerrar.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>38</width>
              <height>38</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_Contenido">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_Control">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>0</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame{
background-color:rgb(0,206,151);
}
QPushButton{
background-color:rgb(61,61,61);
border-top-left-radius:20px;
border-bottom-left-radius:20px;
color:rgb(255,255,255);
font:77 10pt &quot;Arial Black&quot;;
}
QPushButton:hover{
background-color:white;
border-top-left-radius:20px;
border-botto,-left-radius:20px;
color:rgb(0,0,0);
font:77 10pt &quot;Arial Black&quot;;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>API</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>40</height>
                </size>
               </property>
               <property name="text">
                <string>BASE DATOS</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_Pagina">
            <property name="styleSheet">
             <string notr="true">QFrame{
background-color:rgb(61,61,61);
}

QLabel{
font:87 12pt &quot;Arial Black&quot;;
background-color:#000000ff;
color:rgb(0,206,151);
border:0px solid #14C8DC;

}


QLineEdit{
border:0px;
color:rgb(255,255,255);
border-bottom:2px solid rgb(61,61,61);
font:75 12pt &quot;Times New Roman&quot;;
}

QPushButton{
background-color:rgb(61,61,61);
border-radius: 15px;
color:rgb(255,255,255);
font:77 10pt &quot;Arial Black&quot;;

}

QPushButton:hover{
background-color:rgb(0,206,151);
border-radius: 15px;
color:rgb(0,0,0);
font: 77 10pt &quot;Arial Black&quot;;
}

QTableWidget{
background-color: rgb(255,255,255);
color: rgb(0, 0, 0);
gridline-color: rgb(0,206,151);
font-size: 12pt;
color: #000000;
}


QHeaderView::section{
background-color: rgb(0,206,151);
border: 1px solid rgb(0,0,0);
font-size:12pt;


}

QTableWidget QTableCornerButton::section{
background-color:rgb(0,0,0);
border:1px solid rgb(0,206,151);
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>1</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>1</number>
             </property>
             <item>
              <widget class="QStackedWidget" name="stackedWidget">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <widget class="QWidget" name="page_Api">
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTableWidget" name="tableWidget">
                   <column>
                    <property name="text">
                     <string>ID</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>FEC_ALTA</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>USER_NAME</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>CODIGO_ZIP</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>CREDIT_CARD_NUM</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>CREDIT_CARD_CCV</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>CUENTA_NUMERO</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>DIRECCION</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>GEO_LATITUD</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>GEO_LONGITUD</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>COLOR_FAVORITO</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>IP</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>AUTO</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>AUTO_MODELO</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>AUTO_TIPO</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>AUTO_COLOR</string>
                    </property>
                   </column>
                   <column>
                    <property name="text">
                     <string>FEC_BIRTHDAY</string>
                    </property>
                   </column>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <spacer name="horizontalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>88</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <widget class="QPushButton" name="btn_Refrescar">
                     <property name="minimumSize">
                      <size>
                       <width>120</width>
                       <height>30</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Refrescar</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="page_BD"/>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
